src/clientdlg.cpp

```diff
@@ -96,6 +96,15 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
 
     butConnect->setAccessibleName ( tr ( "Connect and disconnect toggle button" ) );
 
+
+    // add to favorites button
+    butAddtoFavorites->setWhatsThis ( "<b>" + tr ( "Add to Favorites Button" ) + ":</b> " +
```

Comment:
`Button` should be lower case "B".

Reply:
This code should not be there. There is no longer a butAddtoFavorites.
What does the "Outdated" at the top mean ?

Reply:
It means a later commit than my comment changed it.